src/Plugin/Field/FieldWidget/OgComplex.php

```diff
@@ -79,4 +97,26 @@ public function elementValidate($element, FormStateInterface $form_state, $form)
 
     // todo: Check the writing permission for the current user.
   }
+
+  /**
+   * {@inheritdoc}
+   */
+  public static function getWidgetState(array $parents, $field_name, FormStateInterface $form_state) {
+    $widgetState = parent::getWidgetState($parents, $field_name, $form_state);
+
+    if (empty(self::$info)) {
+      return $widgetState;
+    }
+    $info = self::$info;
+    $results = \Drupal::entityQuery('og_membership')
```

Comment:
This will just run every time. As self::$info is not assigned to anything, and if `empty(self::$info)` is confusing. Is the static caching needed? This is just for the widget, so this should not be running multiple times per request per widget anyway?